app/assets/stylesheets/templates/_messages.scss

```diff
@@ -2,8 +2,95 @@
   .slab--header {
     padding: {
       top: 1em;
-      bottom: 1em;
+      bottom: .5em;
+    }
+    h2, p {
+    	margin-bottom: 0;
     }
   }
-
 }
+.message--container {
+	display: flex;
```

Comment:
Huh, I think Sublime is set up to do this (Spaces:2) but I'm sure I'm missing something. Would love your 👀 !